classes/image_layer.py

```diff
@@ -108,7 +108,9 @@ def to_dict(self):
             pkg_list.append(pkg.to_dict())
             layer_dict.update({self.fs_hash: {'packages': pkg_list,
                                               'tar_file': self.__tar_file,
-                                              'created_by': self.__created_by
+                                              'created_by': self.__created_by,
+                                              'diff_id': self.__diff_id,
+                                              'import_str': self.__import_str
                                               }})
```

Comment:
Would you mind using self.tar_file etc instead of the private name for the properties? Thanks!

Reply:
yeah ofcourse